packages/@react-aria/interactions/src/usePress.ts

```diff
@@ -740,5 +740,9 @@ function shouldPreventDefault(target: Element) {
 
 function isVirtualPointerEvent(event: PointerEvent) {
   // If the pointer size is zero, then we assume it's from a screen reader.
-  return event.width === 0 && event.height === 0;
+  // Android TalkBack double tap will sometimes return a event with width and height of 1 so we need to check for a specific combo of buttons, type, and pressure.
+  // According to https://w3c.github.io/pointerevents/#dom-pointerevent-pressure, devices that don't support pressure measures will always return .5 if there is an active button
+  // so this check will return false for mouse clicks. Touch events have a pressure of 1 so this will return false as well so TalkBack double tap is the only combo that will
+  // have pressure = 0 and buttons = 1.
+  return (event.width === 0 && event.height === 0) || (event.buttons === 1 && event.type === 'pointerdown' && event.pressure === 0);
```

Comment:
not thrilled about this but this seems like a combination unique to Android talkback double tap. Trying to see if I can repro the TalkBack behavior outside our code

Reply:
also note I've restricted it to `pointerdown`, pressure is 0 for `pointerup` so can't really distinguish between the two. usePress `onPointerUp` uses `state.pointerType` luckily so we get the proper pointer type from detecting the Android TalkBack double tap in `onPointerDown`